client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout.js

```diff
@@ -240,28 +257,107 @@ export default function WPCheckout( {
 	);
 }
 
-const CheckoutSummaryUI = styled( CheckoutSummaryArea )`
+const CheckoutSummaryTitleLink = styled.button`
+	background: ${( props ) => props.theme.colors.background};
+	border-bottom: 1px solid ${( props ) => props.theme.colors.borderColorLight};
+	color: ${( props ) => props.theme.colors.textColor};
+	display: flex;
+	font-size: 16px;
+	font-weight: ${( props ) => props.theme.weights.bold};
+	justify-content: space-between;
+	padding: 20px;
```

Comment:
Sorry, I missed this small alignment issue in my previous review. Can we update it to look like the screen on the right?

![image](https://user-images.githubusercontent.com/6981253/81215069-c4f5af00-8fa6-11ea-9d64-42915804fcf8.png)

I believe the right measurements are `padding: 20px 23px 20px 14px`